src/main/java/org/owasp/validator/html/model/Attribute.java

```diff
@@ -143,6 +143,35 @@ public String matcherRegEx(boolean hasNext){
             }
         }
         return regExp.toString();
+    }
+
+    public static String mergeRelValuesInAnchor(boolean addNofollow, boolean addNoopenerAndNoreferrer, String currentRelValue) {
+        String newRelValue = "";
+        if (currentRelValue == null || currentRelValue.isEmpty()) {
+            if (addNofollow) newRelValue = "nofollow";
+            if (addNoopenerAndNoreferrer) newRelValue += " noopener noreferrer";
+        }
```

Comment:
Combine these two lines into one line:  } else {

Reply:
There is no else. The two cases are independent and can happen both or just one.